packages/cozy-stack-client/src/AppCollection.js

```diff
@@ -0,0 +1,36 @@
+import { uri } from './utils'
+import { normalizeDoc } from './DocumentCollection'
+
+/**
+ * Abstracts a collection of documents of the same doctype, providing
+ * CRUD methods and other helpers.
+ */
+export default class AppCollection {
+  constructor(doctype, client) {
+    this.doctype = doctype
+    this.client = client
+  }
+
+  /**
+   * Lists all documents of the collection, without filters.
+   *
+   * The returned documents are not paginated by the stack.
+   *
+   * @return {{data, meta, skip, next}} The JSON API conformant response.
+   * @throws {FetchError}
+   */
+  async all() {
+    const path = uri`/apps/`
+    try {
+      const resp = await this.client.fetch('GET', path)
+      return {
+        data: resp.rows.map(row => normalizeDoc(row.doc, this.doctype)),
+        meta: { count: resp.total_rows },
+        skip: 0,
+        next: false
+      }
+    } catch (error) {
+      throw error
```

Comment:
I think we can remove the `try/catch` block here, no?

Reply:
I remove it

Reply:
cf. https://repl.it/@enguerran/Throw-Catched-Error